gtfs/spec/en/reference.md

```diff
@@ -612,6 +623,69 @@ Describes levels in a station. Useful in conjunction with `pathways.txt`, and is
 |  `level_index` | Float | **Required** | Numeric index of the level that indicates its relative position. <br><br>Ground level should have index `0`, with levels above ground indicated by positive indices and levels below ground by negative indices.|
 |  `level_name` | Text | Optional | Name of the level as seen by the rider inside the building or station.<hr>_Example: Take the elevator to "Mezzanine" or "Platform" or "-1"._|
 
+### location_groups.txt
+
+File: **Optional**
+
+Primary key (`location_group_id`, `location_id`)
+
+Defines location groups that indicate groups of stops where a rider may request pickup or drop off.
+
+| Field Name | Type | Required | Description |
+| ---------- | ---- | ------------ | ----------- |
+| `location_group_id` | ID | **Required** | Identifies a location group. A location group is a group of stops that together indicate locations where a rider may request pickup or drop off.<br><br> By default, every `stop_id` belongs to a `location_group_id` of the same value. Therefore, it is forbidden to define a `location_group_id` with the same value as a `stop_id`.<br><br>Multiple entries in `location_groups.txt` can have the same `location_group_id`. | 
```

Comment:
I recommend removing "By default, every `stop_id` belongs to a `location_group_id` of the same value. Therefore, it is forbidden to define a `location_group_id` with the same value as a `stop_id`."

We no longer need to establish this restriction since they do not share the `stop_times.stop_id` field anymore.

Reply:
Commit pushed - https://github.com/google/transit/pull/388/commits/6314252ed5c422e6d229654dfac282a3c5ca0ce3
Thanks!